plugins/MobileMessaging/SMSProvider/Clockwork.php

```diff
@@ -53,14 +53,25 @@ public function getDescription()
 			';
     }
 
-    public function verifyCredential($apiKey)
+    public function getCredentialFields()
     {
-        $this->getCreditLeft($apiKey);
+        return array(
```

Comment:
Can we move this as default to SMS Provider base class? Then we likely wont break other plugins like http://plugins.piwik.org/FreeMobileMessaging